ambry-frontend/src/main/java/com/github/ambry/frontend/AmbrySecurityService.java

```diff
@@ -378,9 +379,11 @@ private void setCacheHeaders(RestRequest restRequest, RestResponseChannel restRe
       throws RestServiceException {
     Container container = RestUtils.getContainerFromArgs(restRequest.getArgs());
     if (container.isCacheable()) {
+      long cacheTtl = container.getCacheTtlInSecond() != null ? container.getCacheTtlInSecond().longValue()
```

Comment:
QQ: Would it be safer to check for some minimum cache TTL value set by user? Since the requests could be "internet" facing urls, if a cache value has been mistakenly set to a very short value by the user, it could result in lot of requests coming to Ambry.

Reply:
changing cache ttl for container will be an admin op. we should be good.